go/src/koding/db/mongodb/modelhelper/deletedmember.go

```diff
@@ -0,0 +1,78 @@
+package modelhelper
+
+import (
+	"koding/db/models"
```

Comment:
I dont remember when i last compiled goimports :) After upgrading Go in our repo, i could do a pass in our app directory with new binary